go.mod

```diff
@@ -25,6 +25,7 @@ require (
 	github.com/stretchr/testify v1.4.0 // indirect
 	golang.org/x/net v0.0.0-20191004110552-13f9640d40b9 // indirect
 	golang.org/x/sys v0.0.0-20190826190057-c7b8b68b1456 // indirect
+	k8s.io/api v0.16.4
```

Comment:
❤️ for using modules. Fancy!

It's true the future is not evenly distributed.

Reply:
Thanks to @TheoBrigitte and @mig4, because this shit is light years beyond me :D